doc/mongoc_apm_command_failed_get_reply.rst

```diff
@@ -0,0 +1,31 @@
+:man_page: mongoc_apm_command_failed_get_reply
+
+mongoc_apm_command_failed_get_reply()
+========================================
+
+Synopsis
+--------
+
+.. code-block:: c
+
+  const bson_t *
+  mongoc_apm_command_failed_get_reply (
+     const mongoc_apm_command_failed_t *event);
+
+Returns this event's reply. The data is only valid in the scope of the callback that receives this event; copy it if it will be accessed after the callback returns.
```

Comment:
Please mention that this is the server's reply to a command that failed, it contains details about why the command failed. If no server reply was received (for example, if there was a network error) then the reply is a valid empty BSON document.